src/pages/index.tsx

```diff
@@ -43,7 +43,13 @@ export default function Page() {
           the cloud for authentication, storage, APIs, and more.
         </Text>
         <Flex className="home-cta">
-          <Button variation="primary" size="large" gap="small">
+          <Button
+            as="a"
+            href="/javascript/how-amplify-works"
```

Comment:
Sorry, I haven't been keeping up with this page and so I'm wondering if this is supposed to be hardcoded to javascript?

Reply:
Because it's from the /pages/ route, js is the default. If you click the button from any of the /pages/[platform], it'll show correctly. I matched the hardcoding here to the Get Started button.

Reply:
Got it, thank you for confirming!